src/SessionInstance.php

```diff
@@ -69,7 +75,12 @@ protected function init()
 
         session_name($this->name);
 
-        session_start();
+        if ($this->sessionID) {
+            session_id($this->sessionID);
+        }
+
+        # Start the session up, but ignore the error about headers already being sent
+        @session_start();
```

Comment:
Why would you need to create an instance multiple times, it would always refer to the same thing. Can you put some example code in a gist to demonstrate?